adr/0006-retrieving-data-from-find.md

```diff
@@ -0,0 +1,60 @@
+# 2. Retrieving Data from Find Service
+
+Date: 2019-10-15
+
+## Status
+
+Accepted
+
+## Context
+We are currently working towards meeting these two main user needs:
+-   Candidates want to apply to teacher training courses.
+-   Training providers want to view applicants who have applied to their courses.
+
+In order to do this the Apply service needs to retrieve data for `Training Providers`, 
+`Courses` and `Sites` from the Find service.
+
+The main areas where the Apply Service need data from the Find Service are: 
+-   The Apply Service Start Page.
```

Comment:
Users arrive with course info from Find in their URL, and Apply needs to be able to look up the details for this course.